test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -268,11 +268,6 @@
       <Name>DynamoUtilities</Name>
       <Private>False</Private>
     </ProjectReference>
-    <ProjectReference Include="..\..\src\Libraries\PythonNodeModelsWpf\PythonNodeModelsWpf.csproj">
```

Comment:
This is removed intentionally because I find DynamoCore tests should not depend on any wpf lib